packages/data/src/store/selectors.js

```diff
@@ -69,3 +69,15 @@ export function hasFinishedResolution( state, reducerKey, selectorName, args = [
 export function isResolving( state, reducerKey, selectorName, args = [] ) {
 	return getIsResolving( state, reducerKey, selectorName, args ) === true;
 }
+
+/**
+ * Returns the list of the cached resolvers.
+ *
+ * @param {Object} state      Data state.
+ * @param {string} reducerKey Registered store reducer key.
+ *
+ * @return {Object} Resolvers mapped by args and selectorName.
+ */
+export function getCachedResolvers( state, reducerKey ) {
+	return get( state, reducerKey, {} );
```

Comment:
This should actually be covered by this ESLint rule, but is not because it's not a `Literal` type (needs improvement separately):

https://github.com/WordPress/gutenberg/blob/a696e508ad5d3566b447fc48f355e91953a17c4a/.eslintrc.js#L116-L144

This could possibly unexpected behaviors if passing dot-delimited string as `reducerKey`.

Also doesn't seem very necessary, vs. less Lodash-y implementation:

```js
return state.hasOwnProperty( reducerKey ) ? state[ reducerKey ] : {};
```

Reply:
Aside: Looking into this, the improvements are blocked by an upstream bug I'd forgotten about: https://github.com/estools/esquery/issues/67